src/main/java/Todos.java

```diff
@@ -0,0 +1,22 @@
+public class Todos {
+
+    public String item;
+    public boolean marked;
+
+    public Todos(String name, boolean marked) {
+        this.item = name;
+        this.marked = marked;
+    }
+
+    public void setMark() {
+        this.marked = true;
+    }
+
+    public void unMark() {
+        this.marked = false;
+    }
+
+    public boolean is_marked() {
```

Comment:
Please consider writing booleans as booleans ie Instead of is_marked(), can write isMarked